### PR TITLE
Add nanoflann version 1.5.4

### PR DESCRIPTION
**Pull request by d-vo**
Add [nanoflann](https://github.com/jlblancoc/nanoflann) as module.

### CODE REVIEW

File: modules/nanoflann/1.5.4/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module(
+    name = "nanoflann",
+    version = "1.5.4",
+    compatibility_level = 1,
+)
```
Comment (lalten) [inline modules/nanoflann/1.5.4/MODULE.bazel:5]:
```suggestion
)
bazel_dep(name = "rules_cc", version = "0.0.9")
```

File: modules/nanoflann/1.5.4/source.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/jlblancoc/nanoflann/archive/refs/tags/v1.5.4.tar.gz",
```
Comment (lalten) [inline modules/nanoflann/1.5.4/source.json:2]: Doesn't this need something like
```suggestion
    "url": "https://github.com/jlblancoc/nanoflann/archive/refs/tags/v1.5.4.tar.gz",
    "strip_prefix": "nanoflann-v1.5.4",
```

Comment (d-vo) [reply modules/nanoflann/1.5.4/source.json:2]: You are absolutely right. This got lost.... Should be without the `v`, though. Will fix

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (nanoflann) have been updated in this PR. Please review the changes.

File: modules/nanoflann/1.5.4/source.json (hunk 2 above)
Comment (d-vo) [reply modules/nanoflann/1.5.4/source.json:2]: Fixed remaining issues. Thanks!

File: modules/nanoflann/metadata.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/jlblancoc/nanoflann",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```
Comment (Wyverald) [inline modules/nanoflann/metadata.json:5]: mind adding yourself as a maintainer?

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**